### ISSUE TITLE
Actions on turn 2 during raid battles inconsistent

### ISSUE THREAD

**Issue by BalletDuckNinja** [Fate-Grand-Automata/FGA#1720]
### Preparation

- [X] I tested the [latest release](https://github.com/Fate-Grand-Automata/FGA/releases/tag/canary)
- [X] I looked at other issues (even the [closed ones](https://github.com/Fate-Grand-Automata/FGA/issues?q=is%3Aissue+is%3Aclosed))
- [X] I read the [Troubleshooting Guide](https://github.com/Fate-Grand-Automata/FGA/wiki/Troubleshooting)

### FGO server

NA

### FGA build number

2288

### Describe the bug

Often on the second turn of raids, FGA is for some reason unable to perform actions it's supposed to do and defaults to pressing the attack button. I believe this is due to to the popup showing 'thanks to masters, X number of raid boss has been killed' that messes up FGA's recognition of buttons that makes it panic. It seems to keep happening when I make it try to use plugsuit swap after the first turn of raids, but everything is affected, including using skills from servants in the third row. It sometimes works so it's not a problem with the inputted commands, like the difference between NP into next wave vs NP into same wave.

### Video

https://youtu.be/t4zOBwKIVFs

### Device model

Samsung galaxy tab s8

### Android version

12

### Screen size

2560 x 1600 

### RAM

8

**Comment by BalletDuckNinja** [comment]
Android version is actually 13, that was a typo. I've tired to set the turn wait time to 2000ms (maximum) which seems to help turn 2 commands work more often

**Comment by Jeff-M-Hsu** [comment]
Issue still occurs after the turn delay

**Comment by Dudi4PoLFr** [comment]
Can confirm the new version (2305) works perfectly on :

Device model: Samsung Galaxy Fold 5
Android version: 14
Screen size: 1812 x 2176 (inside display)
RAM: 12GB

Thank you for your work,
A very tired and sleepy European.

**Comment by HydeoMajima** [comment]
Stage counter similarity at 100% fixed the issue for me. Didnt try the new versions yet or "next turn in same wave" so check if any of this works

**Comment by BlackGhost6200** [comment]
I was having the same problem.
After using next turn in same wave instead of next turn, It seems to have solved the problem.

**Comment by TroyBaconator** [comment]
Yes all fixed over here as well
Galaxy S22 Ultra

**Comment by captainohcaptain** [comment]
It worked for a while, then reverted back after I adjusted the delay per turn.

How many seconds to set for that?

**Comment by captainohcaptain** [comment]
Already set delay at 10 s and still the issue pops up now and then.

**Comment by captainohcaptain** [comment]
![Screenshot_2023-12-26-19-22-57-34_e9dd5248539caae09fa1e8ca83b8fac7](https://github.com/Fate-Grand-Automata/FGA/assets/133630466/d6c6a56f-3dbf-45c4-8db8-ce0e226b07f8)
![Screenshot_2023-12-26-19-20-10-34_e9dd5248539caae09fa1e8ca83b8fac7](https://github.com/Fate-Grand-Automata/FGA/assets/133630466/01b406d1-592a-4375-8d54-64d172c1fb3d)

**Comment by Konahrik13** [comment]
I have also been having this problem despite trying to change any delays in the fine tune sections. It seemed to be fixed after changing to Next Turn in Same Wave but it kept messing up at the exact same place on the second turn in the second or so run. One thing I have noticed is that the Battle number in the top right keeps changing to 2 after redoing the commands, though I don't know what that means.

Phone details are a Galaxy A52 5G and Android version 13.
The build is 2288 from installing from Google Play.
The commands I have inputted are: >a b d1 e1 f g1 g i1 x21 d e4 f1 64, 4, 4,#,

**Comment by Konahrik13** [comment]
I was running it at 95% and changed it to 100%, still has the same problem at that exact turn.

**Comment by Rockman3x3** [comment]
I've tried everything, especially the alpha build but no matter what I do, I cant get a consistent setup past turn 2. Delay is 10 seconds, I finetuned 2000ms for a turn, Stage counter similarity 100%, nothing has worked.

**Comment by Konahrik13** [comment]
Downloaded the alpha, turned on the raid choice with the default 3 delay, also have the stage similarity at 100% with the same setup I posted above. Attack cards instead of NP being picked keeps on happening on the second turn.

**Comment by Rockman3x3** [comment]
Ive actually found a solution. So I reset EVERYTHING to default, changed raid delay back to 3 secs. This time I turned on "Rendering Settings in High Speed l" to high quality and it just works now
![Screenshot_20231226_071533_FateGO](https://github.com/Fate-Grand-Automata/FGA/assets/154867307/fa12a81f-c365-40b8-9f59-94ae952cbbbb)

**Comment by Rockman3x3** [comment]
Current config to confirm above statement
![Screenshot_20231226_071810_FGA (CI)](https://github.com/Fate-Grand-Automata/FGA/assets/154867307/9636bf4e-1ae5-4e9b-8340-fff61a5dcc77)

**Comment by Konahrik13** [comment]
Already had that ingame setting at that but I reset everything and put it back. Still having the same problem.

**Comment by captainohcaptain** [comment]
> Ive actually found a solution. So I reset EVERYTHING to default, changed raid delay back to 3 secs. This time I turned on "Rendering Settings in High Speed l" to high quality and it just works now ![Screenshot_20231226_071533_FateGO](https://private-user-images.githubusercontent.com/154867307/292867008-fa12a81f-c365-40b8-9f59-94ae952cbbbb.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM1OTc5OTgsIm5iZiI6MTcwMzU5NzY5OCwicGF0aCI6Ii8xNTQ4NjczMDcvMjkyODY3MDA4LWZhMTJhODFmLWMzNjUtNDBiOC05ZjU5LTk0YWU5NTJjYmJiYi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyNlQxMzM0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNzFkMDkyMTMwYmY3ODNlNWJhNzhhMDJkZjZjOWRjZTFlM2FhYWFlYzM0YmI2Mzg5YTMyOWE3Njg3NTZjNzc3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7cF7_5QNfnGqge4ItlYTwgD0Jziu2P1xVLNczYAcK9E)

Tried this. Currently raid attack is ongoing. Will update later if issue reoccurs.

**Comment by starstorm91** [comment]
Build 2306

Device model: Samsung Galaxy S21 FE
Android version: 14

Had similarity at 100%, have delay at 3s, have "next turn in the same wave" and did the game options settings as above with low res and HQ. I don't know why but setting similarity at 100% made this even worse, every time at 2 turn FGA just did attacks and no skills. After resetting similarity to default, things are ok so far.

I'll update if it happens again.

**Comment by reconman** [comment]
> Build 2306
> 
> Device model: Samsung Galaxy S21 FE Android version: 14
> 
> Had similarity at 100%, have delay at 3s, have "next turn in the same wave" and did the game options settings as above with low res and HQ. I don't know why but setting similarity at 100% made this even worse, every time at 2 turn FGA just did attacks and no skills. After resetting similarity to default, things are ok so far.
> 
> I'll update if it happens again.

Use "Next wave" with stage similarity = 100 %.

**Comment by starstorm91** [comment]
> > Build 2306
> > Device model: Samsung Galaxy S21 FE Android version: 14
> > Had similarity at 100%, have delay at 3s, have "next turn in the same wave" and did the game options settings as above with low res and HQ. I don't know why but setting similarity at 100% made this even worse, every time at 2 turn FGA just did attacks and no skills. After resetting similarity to default, things are ok so far.
> > I'll update if it happens again.
> 
> Use "Next wave" with stage similarity = 100 %.

Already had "next wave", also already tried stage similarity = 100 %, but it always resulted in FGA using normal attacks on turn 2. Tried it again, same thing happened again.

**Comment by starstorm91** [comment]
Ok, "next wave" with stage similarity = 100% works. If it's "same wave" it goes to normal attacks on turn 2.

**Comment by Konahrik13** [comment]
> Ok, "next wave" with stage similarity = 100% works. If it's "same wave" with stage similarity = 100% instead it goes to normal attacks on turn 2.

Yeah this seems to be working. Three runs so far and no problems.

**Comment by boingman** [comment]
> Ok, "next wave" with stage similarity = 100% works. If it's "same wave" with stage similarity = 100% instead it goes to normal attacks on turn 2.

Thanks, that seems to have fixed it for me.
